bundles/org.openhab.binding.danfossairunit/src/main/resources/OH-INF/thing/thing-types.xml

```diff
@@ -50,7 +50,10 @@
 			<channel id="current_time" typeId="currentTime"/>
 			<channel id="mode" typeId="mode"/>
 			<channel id="manual_fan_step" typeId="manualFanStep"/>
-			<channel id="manual_fan_speed" typeId="manualFanSpeed"/>
+			<channel id="manual_fan_speed" typeId="manualFanStep">
```

Comment:
I heard that changing channel type is a problem for users defining things using MainUI. The channel will apparently no more appear if the channel type does no more exist.

Reply:
I don't think it's a problem, but even if it would be, it doesn't apply here, since type was already changed in #11147 which was also merged into 3.2. So this is actually changing back, so will be even safer.

Reply:
Sorry, no, it's changed. I'll verify later if it's a problem, but I don't think so.

Reply:
Previous version:
![image](https://user-images.githubusercontent.com/19519842/145061995-3f2e8856-ead5-4ec4-9e4e-c8d9c337a06a.png)

Updating to new version without restarting openHAB:
![image](https://user-images.githubusercontent.com/19519842/145062075-20131225-ca31-4e25-9a39-72c09cded4ce.png)

At this point you are partly right, it's missing in the UI. Then after openHAB restart:
![image](https://user-images.githubusercontent.com/19519842/145062201-47454b84-bd25-477a-979c-851b014f629b.png)

And checked that it's behaving correctly. So you are right, but problem resolves itself after a restart. To me this is acceptable behaviour in exchange for getting rid of the old channel.